jlm/tooling/CommandGraphGenerator.cpp

```diff
@@ -275,7 +291,7 @@ JhlsCommandGraphGenerator::GenerateCommandGraph(const JhlsCommandLineOptions & c
       auto & parserNode = ClangCommand::CreateParsingCommand(
           *commandGraph,
           compilation.InputFile(),
-          tmp_folder.to_str() + CreateParserCommandOutputFile(compilation.InputFile()).to_str(),
+          CreateParserCommandOutputFile(tmp_folder, compilation.InputFile()).to_str(),
```

Comment:
Line 265/281
Using "/tmp/" for the tmp_folder, should it not be std::filesystem::temp_directory_path().string()

Reply:
The entire tmp_folder creation is messy and should not be part of this function in the first place.

Reply:
Fixed